service/application.go

```diff
@@ -139,7 +141,11 @@ func New(params Parameters) (*Application, error) {
 	parserProvider := params.ParserProvider
 	if parserProvider == nil {
 		// use default provider.
```

Comment:
application.go needs the parser provider extension interfaces and the public errors from configsources